subsys/net/lib/app/server.c

```diff
@@ -287,6 +314,23 @@ static inline void new_server(struct net_app_ctx *ctx,
 #endif /* CONFIG_NET_DEBUG_APP */
 }
 
+static struct net_context *find_net_ctx(struct net_app_ctx *ctx,
+					int *idx)
+{
+	int i;
+
```

Comment:
perhaps initialize *idx to -1 in case we return NULL later?

Reply:
Why, if NULL is returned, *idx is not valid, so nobody should be using it in this case.

Reply:
As Paul mentioned, initializing "i" in this case is not really needed.

Reply:
@jukkar I wasn't referring to "i", I was referring to the initial value of *idx when this function is called.  However, @pfalcon made a good point that if we return NULL, we shouldn't use the value of *idx anyway.  So my comment here may be moot.